packages/forma-36-website/src/components/DocFormatter.js

```diff
@@ -0,0 +1,113 @@
+  `,
+  buttonLeft: css`
+    margin-right: 0;
+    border-top-right-radius: 0;
+    border-bottom-right-radius: 0;
+  `,
+  buttonRight: css`
+    margin-left: 0;
+    border-top-left-radius: 0;
+    border-bottom-left-radius: 0;
+    margin-right: ${tokens.spacingM};
+  `,
+  imageLink: css`
+    font-size: ${tokens.fontSizeM};
+    color: ${tokens.colorTextMid};
+    display: inline-flex;
+    align-items: center;
+    text-decoration: none;
+    padding: calc(${tokens.spacing2Xs}/2) ${tokens.spacingXs};
+    border-radius: calc(${tokens.fontSizeS} / 5);
+
+    &:hover {
+      background-color: ${tokens.colorElementLight};
+    }
+
+    img {
+      margin-right: ${tokens.spacingXs};
+    }
+  `,
+  buttonList: css`
+    display: flex;
+  `,
+};
+
+const DocFormatter = ({ frontmatter, children }) => {
+  const [ activeSection, setActiveSection ] = useState('design');
+  const designDocs = Array.isArray(children) && children.filter(child => child.props.mdxType === 'DesignDocs');
+  const developDocs = Array.isArray(children) && children.filter(child => child.props.mdxType === 'DevelopDocs');
+  const hasDevelopDocs = developDocs.length > 0;
+  const hasDesignDocs = designDocs.length > 0;
+
+  const data = {
+    title: frontmatter && frontmatter.title,
+    githubUrl: frontmatter && frontmatter.storybook,
+    storybookUrl: frontmatter && frontmatter.github
+  }
+
+  return (
+    <React.Fragment>
+      {/* Remove styles condition once all docs follow the same structure */}
+      <header css={data.title ? styles.header : ''}>
+        {data.title && <DisplayText>{data.title}</DisplayText>}
+        <div css={styles.buttonList}>
+          {hasDevelopDocs && hasDesignDocs && (
+            <>
+              <Button 
+                size='small'
+                buttonType={activeSection === 'design' ? 'primary' : 'muted'}
+                css={styles.buttonLeft} 
+                icon={activeSection === 'design' && 'CheckCircleTrimmed'} 
+                isActive={activeSection === 'design'} 
+                onClick={() => setActiveSection('design')}>Design</Button>
+              <Button 
+                size='small' 
+                css={styles.buttonRight} 
+                buttonType={activeSection === 'develop' ? 'primary' : 'muted'}
+                icon={activeSection === 'develop' && 'CheckCircleTrimmed'} 
+                isActive={activeSection === 'develop'} 
+                onClick={() => setActiveSection('develop')}>Develop</Button>
+            </>
+          )}
+          
+          {data.githubUrl && (
+            <a css={styles.imageLink} href={data.githubUrl} title={`View ${data.title} in Storybook`}>
+              <img src={storybookIcon} alt="" />
+              <span>Storybook</span>
+            </a>
+          )}
+
+          {data.storybookUrl && (
+            <a css={styles.imageLink} href={data.storybookUrl} title={`View ${data.title} on GitHub`}>
+              <img src={githubIcon} alt="" />
```

Comment:
@burakukula @m10l Yeah, the alt tag here should be empty as it's a decorative image, this is according to [WCAG](https://www.w3.org/WAI/tutorials/images/decorative/). Adding an alt content i.e "Storybook", in this case it will force screen readers to read "Button, Storybook, Storybook".